sentinelsat/sentinel.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, user, password, api_url='https://scihub.copernicus.eu/apihub/
         self.page_size = 100
         self.user_agent = 'sentinelsat/' + sentinelsat_version
         self.session.headers['User-Agent'] = self.user_agent
+        # Can be modified by the user, e.g. to set to {'disable': True}
+        self.progressbar_opts = {}
```

Comment:
Could you please add this to the docstring? Otherwise it will not appear in the documentation, that these are `tqdm` options and that the progressbar can be disabled.